plugins/janus_streaming.c

```diff
@@ -5795,6 +5796,7 @@ static void janus_streaming_rtp_source_free(janus_streaming_rtp_source *source)
 	g_free(source->rtsp_url);
 	g_free(source->rtsp_username);
 	g_free(source->rtsp_password);
+	curl_free(source->rtsp_stream_uri);
```

Comment:
If we use `g_strdup`, this should be `g_free`, which will make it consistent with all other strings we keep track of.